### ISSUE TITLE
User-Cluster etcd slow communication + client-certificate verification errors

### ISSUE THREAD

**Issue by erdii** [kubermatic/kubermatic#5278]
**Bug description**
The run-clusters fire alerts due to slow etcd communication since ~2 Weeks now. Example from `#alerting-run-stable`:

> [stable-europe-west1-b] [qbz79xs7jt] EtcdMemberCommunicationSlow
Etcd cluster "etcd": member communication with 7c346215c9e8e9bd is taking 0.202496s on etcd instance etcd-0.

Additionally, they log a lot of certificate errors, and they look suspiciously like they stem from : https://github.com/etcd-io/etcd/issues/9785

```
WARNING: 2020/04/03 06:48:37 grpc: addrConn.createTransport failed to connect to {10.244.22.27:2379 0  <nil>}. Err :connection error: desc = "transport: authentication handshake failed: remote error: tls: bad certificate". Reconnecting...
2020-04-03 06:49:27.860157 I | embed: rejected connection from "127.0.0.1:40208" (error "tls: failed to verify client's certificate: x509: certificate specifies an incompatible key usage", ServerName "")
WARNING: 2020/04/03 06:49:27 grpc: addrConn.createTransport failed to connect to {127.0.0.1:2379 0  <nil>}. Err :connection error: desc = "transport: authentication handshake failed: remote error: tls: bad certificate". Reconnecting...
```

User-Clusters are continuously functional, though!

**Comment by xrstf** [comment]
Do you have https://github.com/kubermatic/kubermatic/commit/18ce718144d227bbf60c9ffffa37108eec7ca0a8 in your version? If not, this bug report might be bogus.

**Comment by erdii** [comment]
since this landed in 2.14.0-rc1: we didn't have this in place when the bug report was created but kamil upgraded run-2 to that version and it still throws alerts like:

```
[alpha-europe-west1-c] [45g6qqv8wp] EtcdMemberCommunicationSlow
Etcd cluster "etcd": member communication with 9e06482f125e5e32 is taking 0.19865600000000005s on etcd instance etcd-1.
```
(from today 9:29 CEST)

---

and the etcd logs still show the same error:

```
2020-05-13 08:20:39.714213 I | embed: rejected connection from "10.244.8.217:42728" (error "tls: failed to verify client's certificate: x509: certificate specifies an incompatible key usage", ServerName "")
WARNING: 2020/05/13 08:20:39 grpc: addrConn.createTransport failed to connect to {10.244.8.217:2379 0  <nil>}. Err :connection error: desc = "transport: authentication handshake failed: remote error: tls: bad certificate". Reconnecting...
2020-05-13 08:20:46.880744 I | embed: rejected connection from "127.0.0.1:45290" (error "tls: failed to verify client's certificate: x509: certificate specifies an incompatible key usage", ServerName "")
WARNING: 2020/05/13 08:20:46 grpc: addrConn.createTransport failed to connect to {127.0.0.1:2379 0  <nil>}. Err :connection error: desc = "transport: authentication handshake failed: remote error: tls: bad certificate". Reconnecting...
```

do we need to regenerate certs manually?

**Comment by xrstf** [comment]
Not sure, but it should not hurt. Maybe restarting the etcd pods is enough already to let them load the (maybe) new cert.
